collectors/COLLECTORS.md

```diff
@@ -534,6 +534,16 @@ default. To use a third-party collector, visit their GitHub/documentation page a
     servers.
 -   [SSH](https://github.com/Yaser-Amiri/netdata-ssh-module): Monitor failed authentication requests of an SSH server.
 
+If you wanted to install a third party collecotr into your Netdata agent you would do something like this (using [hamedbrd/netdata/tree/logparser](https://github.com/hamedbrd/netdata/tree/logparser) as an example):
```

Comment:
Not really a good example, it is a Netdata fork branch that can be deleted at any time. Wording it differently - an example of a specific use case.